.gitattributes

```diff
@@ -0,0 +1,8 @@
+wct-core/etc/photoshop export-ignore
+wct-core/src/test export-ignore
```

Comment:
I think we still want to keep `wct-core/src/test` because if we exclude it, we can't do a full build with tests, which is part of ensuring that everything is building correctly.

Reply:
Sure, you know best. :+1: